libbeat/cmd/export/export.go

```diff
@@ -71,18 +71,15 @@ func newFileClient(dir string, ver string) (*fileClient, error) {
 	if err != nil {
 		return nil, err
 	}
-	err = os.MkdirAll(path, os.ModePerm)
-	if err != nil {
-		return nil, err
-	}
+	fmt.Println(fmt.Sprintf("Writing to directory %s", path))
 	return &fileClient{ver: *common.MustNewVersion(ver), dir: path}, nil
 }
 
 func (c *stdoutClient) GetVersion() common.Version {
 	return c.ver
 }
 
-func (c *stdoutClient) Write(_ string, body string) error {
+func (c *stdoutClient) Write(_ string, _string, body string) error {
```

Comment:
This compiles by accident (introduces parameter _string of type string) :)

Alternative syntax: `Write(_, _, body string) error`